servicer.go

```diff
@@ -0,0 +1,121 @@
+package ripple_go_sdk
+
+import (
+	"bytes"
+	"encoding/json"
+	"github.com/n-shaburoff/ripple-go-sdk/resources"
+	"github.com/pkg/errors"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+)
+
+const (
+	AuthUrl = "https://auth.eu.prod.ripplenet.net"
+	BaseUrl = "https://rfc.test.ripplexcurrent.com"
+)
+
+type Service interface {
+	Authorize(data resources.Authorization) error
+	Resolve(path string) string
+	Get(path string) ([]byte, error)
+	Post(data interface{}, path string) ([]byte, error)
+}
+
+type servicer struct {
+	http        *http.Client
+	url         *url.URL
+	accessToken string
+}
+
+func (c *servicer) Authorize(data resources.Authorization) error {
```

Comment:
What if token becomes invalid? i do not see any logic for token auto-requesting during some time....